tests/foreman/ui/test_adusergroup.py

```diff
@@ -325,256 +218,6 @@ def test_negative_create_external_with_invalid_name(self):
             ))
             self.assertIsNone(self.usergroup.search(self.usergroup_name))
 
-    @stubbed()
-    @tier2
-    def test_positive_delete_external(self):
-        """Delete External AD User Group
-
-        :id: 364e9ddc-4ab7-46a9-b52c-8159aab7f811
-
-        :Steps:
-
-            1. Create an UserGroup.
-            2. Assign some roles to UserGroup.
-            3. Create an External AD UserGroup as per the UserGroup name in AD.
-            4. Delete the External AD UserGroup.
-               Note:- Deletion as of sat6.1 is possible only via CLI and not
```

Comment:
removed. it's 6.5 and deletion is still not there :(

Reply:
I would keep that stub as that decision is pretty much out of scope for current effort you are working on

Reply:
ok, returned it back